variables.tf

```diff
@@ -76,7 +76,7 @@ variable "number_of_workers" {
 
 variable "python_version" {
   type        = string
-  default     = "3.9"
+  default     = "3"
   description = "The Python version (2, 3 or 3.9) being used to execute a Python shell job"
```

Comment:
Description doesn't match?

Reply:
It accepts all 3 values however it depends on the `command_name` variable. Since the default for `command_name` variable is glueetl so used the default value as 3 for `python_version`.